commitizen/commands/init.py

```diff
@@ -0,0 +1,107 @@
+import questionary
+
+from commitizen import factory, out
+from commitizen.cz import registry
+from commitizen.config import BaseConfig, TomlConfig, IniConfig
+from commitizen.git import get_latest_tag, get_all_tags
+from commitizen.defaults import config_files
+
+
+class Init:
+    def __init__(self, config: BaseConfig, *args):
+        self.config: BaseConfig = config
+        self.cz = factory.commiter_factory(self.config)
+
+    def __call__(self):
+        values_to_add = {}
+
+        # No config file exist
+        if not self.config.path:
+            config_path = self._ask_config_path()
+
+            if "toml" in config_path:
+                self.config = TomlConfig(data="", path=config_path)
+            else:
+                self.config = IniConfig(data="", path=config_path)
+
+            self.config.init_empty_config_file()
+
+            values_to_add["name"] = self._ask_name()
+            tag = self._ask_tag()
+            values_to_add["version"] = tag
```

Comment:
The `tag` should be `tag.strip()` to remove the new lines gotten from git. And actually the version should be parsed to retrieve the version only part of the tag. 
I've tried it and the output of version in my case was `v1.10.1` and it should be `1.10.1`.

You can fix it by doing:

```py
from packaging.version import Version
v = Version(tag.strip())
v.public
```

Reply:
Thanks! I didn't notice that. I move the `strip()` into `git.py` so that the function can get the correct tag, and handle the `Version(...)` part in the `init.py` so that we can get the correct version from a tag.